lib/generators/doorkeeper/migration_generator.rb

```diff
@@ -12,4 +12,11 @@ def install
   def self.next_migration_number(dirname)
     ActiveRecord::Generators::Base.next_migration_number(dirname)
   end
+
+  private
+
+  def resource_owner_name
+    Doorkeeper.configuration.resource_owner_from_credentials.class.
+      name.demodulize.underscore
+  end
```

Comment:
In the dummy app I got `proc_id`. I'm not sure that we can do this in a general and precise way for every application. I would leave this field without FK as before, and leave a note in the README suggesting users to add it by themselves.